app/panel/components/Summary.jsx

```diff
@@ -71,24 +71,67 @@ class Summary extends React.Component {
 	/**
 	 * Lifecycle event
 	 */
-	componentDidMount() {
-		this._setTrackerLatency(this.props);
-		this._updateSiteNotScanned(this.props);
+	static getDerivedStateFromProps(nextProps) {
+		const trackerLatencyTotal = Summary._computeTrackerLatency(nextProps);
+		const disableBlocking = Summary._computeSiteNotScanned(nextProps);
 
-		this._dynamicUIPort = this.context;
-		this._dynamicUIPort.onMessage.addListener(this.handlePortMessage);
-		this._dynamicUIPort.postMessage({ name: 'SummaryComponentDidMount' });
+		// Set page title for Firefox for Android
+		window.document.title = `Ghostery's findings for ${nextProps.pageUrl}`;
+
+		return { trackerLatencyTotal, disableBlocking };
+	}
+
+	/**
+	 * Calculates total tracker latency and sets it to state
```

Comment:
Comment should be updated to reflect that function no longer sets state

Reply:
Nice catch, I usually updated the comments.
Updated on feature/cleanup-cleanup.

Reply:
lgtm in `feature/cleanup-cleanup`